be/src/olap/rowset/segment_v2/page_builder.h

```diff
@@ -79,6 +84,12 @@ class PageBuilder {
     // and should be followed by reset() before reuse the builder
     virtual void release() = 0;
 
+    // Return a Slice which represents the encoded data of current page,and release the slice
+    // caller need to delete the returned slice
+    virtual Slice finish_and_release() {
```

Comment:
I think we can merge release to finish?

Reply:
In some situation, caller just copy the page returned by PageBuilder, so there is no need for caller to release the page,Such as in BinaryDictPageBuilder.finish,BinaryDictPageBuilder's _data_page_builder just return data page is ok,because BinaryDictPageBuilder just copy the page and won't maintain the page